src/flows/components/Flow.tsx

```diff
@@ -1,15 +0,0 @@
-import React from 'react'
-
-// Components
-import PipeList from 'src/flows/components/PipeList'
-import {PopupDrawer, PopupProvider} from 'src/flows/context/popup'
-
-// NOTE: requires a FlowProvider and ResultsProvider
-const Flow = () => (
-  <PopupProvider>
-    <PipeList />
-    <PopupDrawer />
-  </PopupProvider>
-)
```

Comment:
any excuse to kill a feature am i right?

Reply:
omg there are a million little files that just link two components